build.gradle

```diff
@@ -74,9 +74,20 @@ subprojects {
             jvmTarget = "1.8"
 
             // Allow use of @OptIn
-            freeCompilerArgs += "-Xopt-in=kotlin.RequiresOptIn"
+            freeCompilerArgs += '-Xopt-in=kotlin.RequiresOptIn'
+        }
+    }
 
-            freeCompilerArgs += "-Xallow-jvm-ir-dependencies"
+    // FIXME: Only needed for Compose 1.0.0-alpha12 snapshots
+    configurations.configureEach {
+        resolutionStrategy.eachDependency { DependencyResolveDetails details ->
+            if (details.requested.group == 'androidx.lifecycle' &&
+                    details.requested.name != 'lifecycle-viewmodel-compose') {
```

Comment:
Only need this if you're using `viewModel()`

Reply:
It seems to also be needed for `androidx.lifecycle:lifecycle-runtime` which appears to be a transitive dependency of MDC and AndroidX Core
```
Could not find androidx.lifecycle:lifecycle-runtime:2.3.0.
Required by:
    project :lib > com.google.android.material:material:1.1.0
    project :lib > androidx.core:core:1.5.0-alpha01
```

Reply:
Ahhhh, fair enough.